conbench/tests/api/_fixtures.py

```diff
@@ -290,7 +293,15 @@ def benchmark_result(
 
     if results is not None:
         unit = unit if unit else "s"
-        data["stats"] = Conbench._stats(results, unit, [], "s", one_sample_no_mean)
+        data["stats"] = validate_and_aggregate_samples(
+            {
+                "data": results,
+                "unit": unit,
+                "times": [],
+                "time_unit": "s",
+                "iterations": len(results),
+            }
+        )
```

Comment:
Notable change; the `Conbench` class got moved to `conbenchlegacy`. This (which writes straight to the db) now uses the same code the API uses which is...better? Less bad?

Reply:
haha wow